bin/setup

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env sh
+
+# Set up Rails app. Run this script immediately after cloning the codebase.
+
+# Exit if any subcommand fails
+set -e
+
+# Set up ruby version file
+if [ ! -f .ruby-version ]; then
+  cp .ruby-version.example .ruby-version
+fi
+
+# Set up configurable environment variables
+if [ ! -f .env ]; then
+  cp config/env .env
+fi
+
+# Set up Ruby dependencies via Bundler
+gem install bundler --conservative
+bundle check || bundle install --path=vendor/bundle
```

Comment:
Vendoring is helpful when developing multiple projects and NOT using containers.
(Whether this is a good idea these days is another topic.)

I'm not opposed to the change, anyone who wants to `vendor` their gems should know how to alter the configuration to do it.